packages/legacy/.eslintrc.cjs

```diff
@@ -8,7 +10,10 @@ module.exports = {
   parserOptions: {
     tsConfigRootDir: __dirname,
     extraFileExtensions: ['.svelte'],
-    project: ['./tsconfig.json', './playground/tsconfig.json'],
+    project: [
+      path.join(__dirname, './tsconfig.json'),
+      path.join(__dirname, './playground/tsconfig.json'),
```

Comment:
These `path.join`'s shouldn't be required; might be related to casing comment above

Reply:
Unfortunately the casing did not affect the need for the `path.join`s. Leaving as-is.